cache/private/private_topic_test.go

```diff
@@ -101,12 +112,24 @@ var (
 		SubtopicIds: &[]string{"1834"},
 	}
 
+	testEconomyRootTopic2 = models.Topic{
```

Comment:
Is this section duplicated in the public_test ?
and the next block of changes ?

Reply:
we've spoken about this.